cerberus/prometheus/client.py

```diff
@@ -23,3 +24,27 @@ def get_metrics(query):
         return prom_cli.custom_query(query=query, params=None)
     except Exception as e:
         logging.error("Failed to get the metrics: %s" % e)
+
+
+# Alert on high latencies
+def display_metrics(distribution, prometheus_url, prometheus_bearer_token):
```

Comment:
Maybe make this a generic function which accepts a prometheus query, runs it and returns the output?  This way we can use it to pass the query of interest ( ALERTS{alertname="KubeAPILatencyHigh", severity="warning"} in this case ) and warn the users with a custom message.

Reply:
Fixed it.